connectors/jetty-connector/src/main/java/org/glassfish/jersey/jetty/connector/JettyConnector.java

```diff
@@ -184,6 +184,13 @@
             final URI u = getProxyUri(proxyUri);
             final ProxyConfiguration proxyConfig = client.getProxyConfiguration();
             proxyConfig.getProxies().add(new HttpProxy(u.getHost(), u.getPort()));
+
+            final Object proxyUsername = config.getProperties().get(ClientProperties.PROXY_USERNAME);
+            if (proxyUsername != null) {
+                final Object proxyPassword = config.getProperties().get(ClientProperties.PROXY_PASSWORD);
+                auth.addAuthentication(new BasicAuthentication(u, "<<ANY_REALM>>",
+                        proxyUsername.toString(), proxyPassword.toString()));
```

Comment:
why not to use ```String.valueOf(proxyPassword)``` here to prevent occasional NPE?

Reply:
You're right. I'm going to change.